osidb/tests/test_flaw.py

```diff
@@ -611,6 +611,26 @@ def test_major_incident_start_dt(self):
             assert f.major_incident_start_dt == timezone.now()
 
 
+class TestFlawComment:
+    def test_multiple_objects_returned(self):
+        """
+        test that the MultipleObjectsReturned case of create_flawcomment is handled well
+        this is the reproducer for https://issues.redhat.com/browse/OSIDB-3086
```

Comment:
Internal URL.

```suggestion
        this is the reproducer for OSIDB-3086
```

Reply:
I already hit auto-merge and went home :smile: I agree that it is internal but I am a bit hesitant to consider it an issue. Red Hat Jira itself is public and issue ID does not hold any information above its existence so putting there only issue ID seems to me as providing completely the same information. Or do you think otherwise? Anyway we have a plenty of such cases throughout the code base.